.github/workflows/scripts/kubernetes/qs-overrides/microprofile-reactive-messaging-kafka/overridable-functions.sh

```diff
@@ -124,11 +124,7 @@ EOF
 # 1 - application name
```

Comment:
just in case, are those changes supposed to be here?

Reply:
Not sure exactly which you mean here, but if you mean changing what I am deleting I found a better way :-)

Reply:
Was just an alert, could have been unintended